features/storage/csi.feature

```diff
@@ -31,3 +31,16 @@ Feature: CSI testing related feature
     Given I switch to cluster admin pseudo user
     Given admin uses the "csihostpath" project
     Given the pod named "my-csi-app" status becomes :running
+
+  # @author chaoyang@redhat.com
+  # @case_id OCP-37827
+  @admin
+  Scenario: CSI images checking in prod registry env
+    Given the master version >= "4.4"
+    Given I switch to cluster admin pseudo user
+    Given admin uses the "csihostpath" project
+    Given the pod named "my-csi-app" status becomes :running
+    And the expression should be true> pvc("csi-pvc").capacity(cached: false) == "2Gi"
+    When I run the :get admin command with:
+      | resource | volumesnapshot |
+    Then the output should contain "true"
```

Comment:
Exactly the same steps as in OCP-30787

Perhaps we can merge them into scenario outline 
```
  # @author chaoyang@redhat.com
  # @case_id OCP-30787
  @admin
  Scenario Outline: CSI images checking
    Given the master version >= "4.4"
    Given I log the message> CSI images checking in <cluster> env
    Given I switch to cluster admin pseudo user
    Given admin uses the "csihostpath" project
    Given the pod named "my-csi-app" status becomes :running
    And the expression should be true> pvc("csi-pvc").capacity(cached: false) == "2Gi"
    When I run the :get admin command with:
      | resource | volumesnapshot |
    Then the output should contain "true"
    Examples:
      | cluster |
      | stage   | # @case_id OCP-30787
      | prod    | # @case_id OCP-37827
```